src/main/java/io/appform/dropwizard/discovery/bundle/resolvers/PortSchemeResolver.java

```diff
@@ -0,0 +1,7 @@
+package io.appform.dropwizard.discovery.bundle.resolvers;
+
+import io.dropwizard.server.ServerFactory;
+
+public interface PortSchemeResolver extends CriteriaResolver<String, ServerFactory> {
```

Comment:
Add Javadoc. Add FunctionalInterface

Reply:
Take `<C extends Configuration>` as function param. so this will become: 
`public interface PortSchemeResolver<C extends Configuration> extends Function<String, C>`  or something like that.

Reply:
Had a criteria resolver thinking if I ever need all criteriaResolvers can fetch them using reflections all at once. No point extending a Function like that, instead can just do a Simple named resolver, PortSchemeResolver with a method getPortScheme(... 

lmk what you think.